project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Add to cart popup visibility is broken
<img width="884" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/8d18144e-a513-4458-8daa-8bc722e8ce21">